Java/ds/Intersection_of_two_linkedlists.java

```diff
@@ -0,0 +1,144 @@
+//given two linked lists. and we have to find intersection oint between these two linkedlist. So Here is the program for finding it.
```

Comment:
This doesn't look very formal. Can you please take a look at existing code and make the description more standard. Also use multi-line comments for the purpose.